bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -1031,6 +1089,9 @@
                 if (specifier.hasOwnProperty('subclasses'))
                     subclasses = specifier.subclasses;
             }
+            if (typeof cls === 'string') {
```

Comment:
It would be good to keep the API as explicit as possible, so that's why I removed this.